### PR TITLE
Magento 107

### PR DESCRIPTION
**Pull request by suraiyavic**
Fix for Open Contract ID and Currency not being Saved properly

### CODE REVIEW

File: Observer/SalesOrderPlaceAfterObserver.php [hunk 1]
```diff
@@ -117,15 +119,22 @@ protected function fetchContractDetail($contractId)
...
-                    $detail['method']=$response['Payment']['Method'];
-                    $detail['identifier']=$response['Payment']['AccountIdentifier'];
+                    $detail['currency']= $response_extracted['ConsumerCurrency'];
+                    $detail['method']=$response_extracted['Payment']['Method'];
+                    $detail['identifier']=$response_extracted['Payment']['AccountIdentifier'];
 
-                    if (isset($response['Times']['Expiry'])) {
-                        $expredAt = explode('T', $response['Times']['Expiry']);
+                    if (isset($response_extracted['Times']['Expiry'])) {
+                        $expredAt = explode('T',$response_extracted['Times']['Expiry']);
```
Comment (lizzyWithReach) [inline Observer/SalesOrderPlaceAfterObserver.php]: can you fix the expredAt typo here to "expiredAt"?

**Review by lizzyWithReach — CHANGES_REQUESTED** [review]
Just a little typo, if you think it's reasonable to fix that at this time Suraiya. (if it's actually a weed of a typo that exists everywhere then I'm ok leaving it)

File: Observer/SalesOrderPlaceAfterObserver.php (hunk 1 above)
Comment (suraiyavic) [reply Observer/SalesOrderPlaceAfterObserver.php]: Sure Lizzy. Here you go: 298f55e308cde2677e9a1d46ae7ccc0926cd6562

**Review by lizzyWithReach — APPROVED** [review]
Thank you!

**Pull request merged**